spring-profiler-common/src/main/java/io/github/linyimin0812/profiler/common/logger/LogFactory.java

```diff
@@ -13,11 +14,21 @@ public class LogFactory {
     private final static Map<LoggerName, Logger> LOGGER_MAP = new HashMap<>();
 
     static {
+        initialize();
+    }
+
+    static void initialize() {
+        LOGGER_MAP.clear();
         createLogger(LoggerName.STARTUP);
```

Comment:
The startup and transform also need to specify paths; otherwise, the default paths will be used.

```java
 createLogger(LoggerName.STARTUP, logPath);
 createLogger(LoggerName.TRANSFORM, logPath);
```